analyzer/testdata/src/debug/contains_replacements_test.go

```diff
@@ -0,0 +1,70 @@
+package debug
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContainsNotContainsReplacements_String(t *testing.T) {
+	var tm tMock
+
+	const substr = "abc123"
+
+	for _, s := range []string{"", "random", substr + substr} {
+		t.Run(s, func(t *testing.T) {
+			assert.Equal(t,
+				assert.True(tm, strings.Contains(s, substr)),
+				assert.Contains(tm, s, substr),
+			)
+
+			assert.Equal(t,
+				assert.False(tm, !strings.Contains(s, substr)),
+				assert.Contains(tm, s, substr),
+			)
+
+			assert.Equal(t,
+				assert.False(tm, strings.Contains(s, substr)),
+				assert.NotContains(tm, s, substr),
+			)
+
+			assert.Equal(t,
+				assert.True(tm, !strings.Contains(s, substr)),
+				assert.NotContains(tm, s, substr),
+			)
+		})
+	}
+}
+
+func TestContainsNotContainsReplacements_Bytes(t *testing.T) {
```

Comment:
added debug tests to be sure that this is not supported

<img width="800" alt="Screenshot 2024-06-25 at 20 27 35" src="https://github.com/Antonboom/testifylint/assets/17127404/6cecc3bf-164b-4204-b2dd-6d907dfbf404">

Reply:
How do you prevent these failing tests to fail the `go test ./...`

I don't get where is the magic?